zim/gui/pageview/__init__.py

```diff
@@ -6476,8 +6481,8 @@ def _checkbox_do_populate_popup(self, menu, buffer, iter):
 		menu.prepend(Gtk.SeparatorMenuItem())
 
 		for bullet, label in (
```

Comment:
The popup menu is populated in reverse order, which thus corresponds to the toggle order after this change:

```
[ ] 0
[*] 1
[x] 2
[>] 3
[<] 4
```